examples/concertbot/data/rules.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+  - rule: fallback
+    steps:
+      - ...
+      - intent: nlu_fallback
```

Comment:
did you test this works? I personally have no idea how the fallback works in the new setup. Doesn't the threshold need to be defined somewhere?

Reply:
I implemented it based on Tobias' doc https://github.com/RasaHQ/rasa/pull/6195/files#diff-9bbb964215f342154b3f976a61d47d2cR43. Just updated the config file and it's working now